core/password.go

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright 2018 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * @author: Tingyu Zeng, Dell
+ * @version: 1.0.0
+ *******************************************************************************/
+
+package main
```

Comment:
Every file in this service is in the `main` package. There's not a lot of separation here. I don't see any tests either.

Reply:
We can open an issue with tests in the backlog to address it in the next release. Regarding the structure, would like to address them in the next release if we can have some guidelines to follow.